documentation/forsc_server_options.md

```diff
@@ -33,10 +34,17 @@ You do not need approval for stopping and restarting your instance as long as yo
 However, if you require additional time beyond your estimate, please notify us by posting a brief justification to your topic.
 If your analysis exceeds the estimated runtime, we may reach out to discuss potential termination.
 
-## Environment
+(base-environment)=
+## Base Environment
 
 The Fornax Science Console offers a number of software environments to choose from.
-These include:
+They are currently grouped into two base environments.
+You can select a base environment by choosing one of the following container images from the "Environment" dropdown in the screenshot above:
 
--   **Default Astrophysics** *(recommended for most users)* This software environment
--   **High-Energy Astrophysics**
+-   **Default Astrophysics** (recommended for most use cases) contains many common astronomy software, including those required to run the demo notebooks.
+    This container image is referred to as **fornax-main**.
+-   **High-Energy Astrophysics** contains high-energy software, which currently includes HEASoft.
+    Plans exist to add CIAO, XMM-SAS and Fermitools.
+    This container image is referred to as **fornax-hea**.
+
+See {ref}`compute-environments` for more information about the specific software environments that are pre-installed in each container image and how to customize your environment after your server starts up.
```

Comment:
This is note to @bsipocz  to investigate: the ref to base-environment in the previous page gives a nice hover over, and this doesn't. Figure out why.